ironfish/src/memPool/memPool.ts

```diff
@@ -308,6 +311,8 @@ export class MemPool {
       this.metrics.memPoolEvictions.value += evicted.length
     }
 
```

Comment:
Will this emit messages on a valid transaction? Do we want to emit evicted/rejected transactions too?

Reply:
This will only emit for a valid transaction that is successfully added to the mempool.

We could emit for evictions and rejections, but for now I don't know what we'd do with those events